apps/hubble-stats/app/layout.tsx

```diff
@@ -26,13 +27,14 @@ export default function RootLayout({
         <Head>
           <title>Welcome to Hubble Stats!</title>
         </Head>
```

Comment:
We don't need to use the `<Head></Head>` component anymore in the `app` directory. You can export a `metadata` variable to set the [metadata](https://nextjs.org/docs/app/building-your-application/optimizing/metadata) for the page.